test/contracts/common/erc20-transfer.ts

```diff
@@ -0,0 +1,83 @@
+import {ExpectTokenTransfer} from '../bond/single-collateral-bond-events'
+import {ActualTokenTransfer} from '../bond/verify-single-collateral-bond-events'
+import {BigNumber, BaseContract, ContractReceipt, Event} from 'ethers'
+import {TransferEvent} from '../../../typechain/IERC20'
+import {expect} from 'chai'
+import {events} from '../../framework/events'
+import {verifyOrderedEvents} from '../../framework/verify'
+import {eventLog} from '../../framework/event-logs'
+import {Result} from '@ethersproject/abi'
+import {log} from '../../../config/logging'
+
+export function deepEqualsERC20TokenTransfer(
+    actual: ActualTokenTransfer,
+    expected: ExpectTokenTransfer
+): boolean {
+    return (
+        actual.to === expected.to &&
+        actual.from === expected.from &&
+        actual.value.toBigInt() === expected.amount
+    )
+}
+
+/**
+ * Shape check and conversion for a TransferEvents.
+ */
+export function erc20TransferEvents(_events: Event[]): {
```

Comment:
suggestion: drop the underscore prefix for method parameters.

Consistency. Elsewhere you'll find the underscore being used for private members, also used in that manner in the Solidity.

Reply:
Was added for a name clash with an import. The import is now gone so problem solved.